src/containers/Tenants/Tenants.tsx

```diff
@@ -314,3 +279,69 @@ export const Tenants = ({additionalTenantsProps}: TenantsProps) => {
         </div>
     );
 };
+
+function getDBActionsColumn({
+    clusterName,
+    isEditDBAvailable,
+    isDeleteDBAvailable,
+}: {
+    clusterName: string;
+    isEditDBAvailable?: boolean;
+    isDeleteDBAvailable?: boolean;
+}) {
+    return {
```

Comment:
I think, such a column should be sticky.

Reply:
What do you mean by "tricky"?

Reply:
Created an issue for this to discuss and do it later: https://github.com/ydb-platform/ydb-embedded-ui/issues/2310

1. I don't want to add any changes to table while Anton is solving performance issues and doing a lot of tables refactoring - it may cause conflicts
2. I think it should be discussed it with the designer first
3. For our tables it may require some tricky css, because there are no sticky columns in react-data-table by props. So I'd prefer to do it after our planned tasks will be done